### ISSUE TITLE
Use only a single h1 per page 

### ISSUE THREAD

**Issue by MrGreenTea** [YoungVision-eV/website#41]
According to the html spec and best practices every webpage should only have a single `h1` tag.

Most of our pages have multiple `h1` tags though; for each section one. These should be changed to `h2` and the rest of the heading tags will have to be adjusted accordingly (`h3`, `h4`, ...)
